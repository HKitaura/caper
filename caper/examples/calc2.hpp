#include "calc2_ast.hpp"
#include "calc2_parser.hpp"
