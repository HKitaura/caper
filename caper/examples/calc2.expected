expr 7 * 4
expr 3 + 7 * 4
expr 3 + 7 * 4 - 1
accepted
30
